client/src/components/messenger/Chat.js

```diff
@@ -279,11 +279,13 @@ const Chat = ({ conversationId, title, participants, closeChat, isDark }) => {
                             className={"message " + (msg.senderId === id ? "author" : "friend") + "_message"}
                         >
                             <div className="info">
-                                <span className="user" title={msg.sender?.displayName}>{msg.sender?.displayName}</span>
+                                <span className="user" title={msg.sender?.displayName + "#" + msg.senderId}>
+                                    {msg.sender?.displayName}</span>
```

Comment:
shouldn't this be the same as what's in the title?

  `{msg.sender?.displayName + "#" + msg.senderId}</span>`

Reply:
The display name next to the avatar image in the chat is the just the display name, i.e. what you would refer to your friend  as (you wouldn't say Chris#23, irl, just Chris). The # + id is in the title for if you wanted to find out the id fast by hovering. Furthermore with space limits, the full displayname (with id) may not be visible anyway.

Reply:
Ok understood. In this case maybe instead of implementing the other changes I suggested, include a new filed in the contact details card. Incase someone wants to pass on someone else's addable details to a 3rd person?